src/Settings/FlyView.SettingsGroup.json

```diff
@@ -18,5 +18,11 @@
     "shortDescription": "Show/Hide Log Replay status bar",
     "type":             "bool",
     "defaultValue":     false
+},
+{
+    "name":             "alternateInstrumentPanel",
+    "shortDescription": "Use Alternate Instrument Panel instead of the default one",
```

Comment:
This should match the wording show to the user to some extent. So should use the word "Vertical".